src/components/Search/Search.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import SearchButton from './SearchButton.js';
+import TablTitle from './SearchHeading.js';
+import SearchInput from './SearchInput.js';
+
+const Search = props => (
+  <div className="search">
+    <div className="page-header">
+      <h4 className="text-left">Search Bookings</h4>
+    </div>
+    <div className="row">
+      <div className="col"></div>
+      <table className="table search-table">
+        <thead>
+          <tr>
+            <TablTitle title='Customer id' /><th />
+            <TablTitle title='Customer Name' /><th />
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <SearchInput id='customerId' placeholder='Customer id' name='customerid' />
```

Comment:
If you have lots of props, I like to format my code more like this:

```js
<SearchInput
  id='customerId'
  placeholder='Customer id'
  name='customerid'
/>
```

Reply:
Done